packages/typespec-ts/test/integration/generated/special-headers/client-request-id/.eslintrc.json

```diff
@@ -1,14 +0,0 @@
-{
```

Comment:
why this is removed?

Reply:
According to spec , `special-headers/client-request-id` had been moved to `azure/special-headers/client-request-id`, 
I forgot to remove the extra files during the last update. https://github.com/Azure/autorest.typescript/tree/main/packages/typespec-ts/test/integration/generated/azure/special-headers/client-request-id